modules/users/client/views/authentication/signup.client.view.html

```diff
@@ -63,9 +63,7 @@ <h3 class="col-md-12 text-center">Sign Up</h3>
           <progressbar value="strengthProgress" type="{{strengthColor}}"><span style="color:white; white-space:nowrap;">{{strengthProgress}}%</span></progressbar>
         </div>
         <div class="text-center form-group">
-          <button type="submit" class="btn btn-primary">Sign up</button>
-          &nbsp; or&nbsp;
-          <a ui-sref="authentication.signin" class="show-signup">Sign in</a>
+          <button onclick="location.href='/admin/users'" type="submit" class="btn btn-primary">Create</button>
```

Comment:
instead, probably make this call a method onclick that checks if the information is valid